### PR TITLE
WatermarkOffsets .ToString()

### PR DESCRIPTION
**Pull request by mhowlett**

### CODE REVIEW

File: src/Confluent.Kafka/WatermarkOffsets.cs [hunk 1]
```diff
@@ -28,5 +28,8 @@ public WatermarkOffsets(Offset low, Offset high)
 
         public Offset Low { get; }
         public Offset High { get; }
+
+        public override string ToString()
+            => $"[{Low}, {High}]";
```
Comment (edenhill) [inline src/Confluent.Kafka/WatermarkOffsets.cs]: This looks more like a list with elements rather than a range.
Maybe "L-H" or "L..H" ?

Comment (mhowlett) [inline src/Confluent.Kafka/WatermarkOffsets.cs]: or maybe explicitly spelling it out: `Low: <low> High: <high>`
not sure

Comment (ewencp) [inline src/Confluent.Kafka/WatermarkOffsets.cs]: Seems like anything to indicate a range would be fine here -- otherwise this patch LGTM

Comment (mhowlett) [inline src/Confluent.Kafka/WatermarkOffsets.cs]: [L, H] is in fact correct mathematical notation for a closed interval (which this is), though mathematicians are not the audience i guess.

What about:

`<low> -> <high>`

?

I was just thinking we're being somewhat inconsistent with `ToString`. `Metadata` gives a JSON representation of the object (which will generally be quite large). Everything else something custom. I think this is what we want.

Comment (edenhill) [inline src/Confluent.Kafka/WatermarkOffsets.cs]: I'd say .. and - are the most common range separators in computer lingo.

**Comment by mhowlett** [comment]
ok .. is fine by me.

**Pull request merged**